### PR TITLE
Project week 3 - rain check - Sofia W

### PR DESCRIPTION
**Pull request by efstasia**

### CODE REVIEW

File: code/images/about.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This favicon was generated using the following graphics from Twitter Twemoji:
```
Comment (Mattsson717) [inline code/images/about.txt:1]: Nice, now I now where to give credit to copyrighted images!

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,159 @@
...
+    <link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    
+    <title>Rain check</title>
+</head>
+<body>
+    
+  <header>
+    <!--makes it possible to jump to the top-->
+    <div id="top"></div>
```
Comment (Mattsson717) [inline code/index.html:19]: I learned from my review that you can put the id="top" in the header. Dont need to make a div for it :)

File: code/index.html [hunk 3]
```diff
@@ -1,30 +1,159 @@
...
+      <nav class="navigation">
+          <a href="index.html">Home</a>
+          <a href="#join"><i class="fas fa-tint"></i>Join us</a>
+          <a href="user.html"><i class="fas fa-sign-in-alt"></i>Sign in</a> 
+          <a href="#faq"><i class="far fa-question-circle">FAQ</i></a>
+          <a href="#contact"><i class="fas fa-envelope-square">Contact us</i></a>
```
Comment (Mattsson717) [inline code/index.html:25]: I like the icons

File: code/index.html [hunk 4]
```diff
@@ -1,30 +1,159 @@
...
+      </nav>
+        
+      <div class="hero">
+          <video loop muted autoplay>
+              <source src="videos/production ID_5197762.mp4" type="video/mp4">
+              <source src="videos/production ID_5197762.ogg" type="video/mp4">
+          </video>
+          <div class="caption">
+              <h1>Rain<i class="far fa-check-square"></i></h1>
+              <p><em>We might be called rain check, but we believe in delivering when we said we would</em></p>
```
Comment (Mattsson717) [inline code/index.html:35]: what does the <em> tag do? Havent seen that before.

File: code/script.js [hunk 5]
```diff
@@ -1,9 +1,23 @@
...
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+
+document.getElementById('signup').addEventListener('submit', (event) => {
+    event.preventDefault()
+
+    const inputValue = document.getElementById('fname').value
+    const greeting = document.getElementById('greeting')
```
Comment (Mattsson717) [inline code/script.js:15]: Nice adding the greeting :)

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,387 @@
...
+}
+
+.hero {
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+/* a filter that darkens the video */
```
Comment (Mattsson717) [inline code/style.css:48]: Nice

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,387 @@
...
+  height: 100px;
+  color: #e7e0c9;
+   text-align: center;
+   font-family: 'Oswald' , sans-serif;
+  margin: auto;
+  padding-top: 20px;
+  font-size: 0.88em;  
+}
+
+/* accordion */
```
Comment (Mattsson717) [inline code/style.css:220]: I really liked the accordion, nice with the scrollbar

**Review by Mattsson717 — COMMENTED** [review]
Really nice page. 
Looks like you met all the blue requirements and even some of the red and black.
The page is responsive, works on all platforms.
Your code is really well written and easy to understand and follow.
Great job"

File: code/index.html (hunk 2 above)
Comment (efstasia) [reply code/index.html:19]: That sounds like the smart thing to do, thank you!

File: code/index.html (hunk 4 above)
Comment (efstasia) [reply code/index.html:35]: Do you mean the `<em></em>`? It makes the text in it cursive

File: code/script.js (hunk 5 above)
Comment (efstasia) [reply code/script.js:15]: Thank you! :)

File: code/style.css (hunk 7 above)
Comment (efstasia) [reply code/style.css:220]: Thank you very much!

**Pull request closed without merging**